### PR TITLE
FAQ: Why don't you compile Matlab/Python/R/… code to Julia?

### PR DESCRIPTION
**Pull request by stevengj**
This seems to be a FAQ.  e.g.

* https://discourse.julialang.org/t/how-hard-would-it-be-to-implement-numpy-jl-i-e-numpy-in-julia/22080
* https://discourse.julialang.org/t/python-to-julia-transpiler/21473
* JuliaLang/Juleps#55
* https://discourse.julialang.org/t/convert-matlab-code-to-julia-1-0/21529
* http://www.stochasticlifestyle.com/why-numba-and-cython-are-not-substitutes-for-julia/

### CODE REVIEW

**Review by StefanKarpinski — APPROVED** [review]
Great addition to the FAQ!

**Comment by JeffBezanson** [comment]
Yes, great answer.

File: doc/src/manual/faq.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
+
+Julia's advantage is that good performance is not limited to a small subset of “built-in” types and operations, and one can write high-level type-generic code that works an arbitrary user-defined types while remaining fast and memory-efficient.  Types in languages like Python simply don't provide enough information to the compiler to enable similar capabilities, so as soon as you used those languages as a Julia front end you would be stuck.
```
Comment (miguelraz) [inline doc/src/manual/faq.md]: typo: works on* arbitrary

Comment (stevengj) [reply doc/src/manual/faq.md]: fixed.

**Review by ararslan — APPROVED** [review]

**Pull request merged**